docs/contributing.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to Kadok's development
+
+Hi there! We are happy that you would like to contribute to Kadok!
+
+Kadok is an open source discord bot. Following these development guidelines will help to keep the project clean and running. 
+
+## How to report a bug
```

Comment:
side note: we should probably create templates to report bug or ask for new features